packages/devtools_app/lib/src/screens/inspector_v2/widget_properties/properties_view.dart

```diff
@@ -20,26 +24,70 @@ class PropertiesView extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return ValueListenableBuilder<List<RemoteDiagnosticsNode>>(
-      valueListenable: controller.selectedNodeProperties,
-      builder: (context, properties, _) {
-        return Container(
-          margin: const EdgeInsets.all(denseSpacing),
-          child: ListView.builder(
-            itemCount: properties.length,
-            itemBuilder: (context, index) {
-              return PropertyItem(
-                index: index,
-                properties: properties,
-              );
-            },
+    return MultiValueListenableBuilder(
+      listenables: [
+        controller.selectedNodeWidgetProperties,
+        controller.selectedNodeRenderProperties,
+      ],
+      builder: (context, values, _) {
+        final widgetProperties = values.first as List<RemoteDiagnosticsNode>;
+        final renderProperties = values.second as List<RemoteDiagnosticsNode>;
+        return Padding(
+          padding: const EdgeInsets.all(denseSpacing),
+          child: AnalyticsTabbedView(
+            gaScreen: gac.inspector,
+            tabs: [
+              (
+                tab: DevToolsTab.create(
+                  tabName: 'Widget properties',
+                  gaPrefix: 'widgetPropertiesTab',
+                ),
+                tabView: PropertiesTable(properties: widgetProperties),
+              ),
+              if (renderProperties.isNotEmpty)
+                (
+                  tab: DevToolsTab.create(
+                    tabName: 'Render object',
+                    gaPrefix: 'renderObjectTab',
+                  ),
+                  tabView: PropertiesTable(properties: renderProperties),
+                ),
+            ],
           ),
         );
       },
     );
   }
 }
 
+class PropertiesTable extends StatelessWidget {
+  const PropertiesTable({
+    super.key,
+    required this.properties,
+  });
+
+  final List<RemoteDiagnosticsNode> properties;
+
+  @override
+  Widget build(BuildContext context) {
+    final scrollController = ScrollController();
```

Comment:
this is likely leaking resources. Do we need to create a ScrollController object? I think a Scrollbar can be wrapped around the list directly and then the ListView and the Scrollbar will be linked because the ListView is the first scrollable child

Reply:
Oh I remember why I added this. I was seeing this error (same error I'm seeing in the tests):

```
  The following assertion was thrown during a scheduler callback:
  The PrimaryScrollController is attached to more than one ScrollPosition.
  The Scrollbar requires a single ScrollPosition in order to be painted.
  When Scrollbar.thumbVisibility is true, the associated ScrollController must only have one
  ScrollPosition attached.
  If a ScrollController has not been provided, the PrimaryScrollController is used by default on
  mobile platforms for ScrollViews with an Axis.vertical scroll direction.
  More than one ScrollView may have tried to use the PrimaryScrollController of the current context.
  ScrollView.primary can override this behavior.
  
  When the exception was thrown, this was the stack:
  #0      RawScrollbarState._debugCheckHasValidScrollPosition.<anonymous closure> (package:flutter/src/widgets/scrollbar.dart:1501:9)
  #1      RawScrollbarState._debugCheckHasValidScrollPosition (package:flutter/src/widgets/scrollbar.dart:1532:6)
  #2      RawScrollbarState._debugScheduleCheckHasValidScrollPosition.<anonymous closure> (package:flutter/src/widgets/scrollbar.dart:1418:14)
  #3      SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1397:15)
  #4      SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1331:11)
  #5      LiveTestWidgetsFlutterBinding.handleDrawFrame (package:flutter_test/src/binding.dart:1804:13)
  #6      SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:1176:5)
  #10     _invoke (dart:ui/hooks.dart:314:10)
  #11     PlatformDispatcher._drawFrame (dart:ui/platform_dispatcher.dart:419:5)
  #12     _drawFrame (dart:ui/hooks.dart:283:31)
  (elided 3 frames from dart:async)
```

Added back in the scroll controller, but this time made one separate for the render / widget properties table and disposing when the widget is disposed